Add a metatable for vectors (#9612)
minetest/minetest: doc/lua_api.txt

> rMinetest Lua Modding API Reference

Reviewer: unintended change?

Author: Yes.

    {x=num, y=num, z=num}

>     Note: it is recommended to construct a vector rather like this:

Reviewer 2: ```suggestion
    Note: it is recommended to construct a vector using the helper function:
```

Author: Done.
